styles/desktop.css

```diff
@@ -7,22 +7,21 @@
     margin: 16px 0 16px 143px;
   }
 
-  @keyframes rotation {
-    from {
-      transform: rotate(0deg);
-    }
+  .icon-menu {
+    display: none;
+  }
 
-    to {
-      transform: rotate(360deg);
-    }
+  .click-event {
+    display: inline-block;
   }
 
-  .icon-menu {
+  .close-btn {
     display: none;
   }
 
   .menu-lists {
```

Comment:
- [ ] Kindly make sure you stick to the design as much as possible. spacing, color...
As you may notice on small screens there's a scroll which is prompting the user to scroll on the right to see the `hamburger` menu, Kinldy fix that.
`Hint: YOu may consider giving your nav-bar or body a width of 100%`

![image](https://user-images.githubusercontent.com/97100091/210395421-9c65db1e-200f-4240-9c37-7f6574135aba.png)